ProjectManager/ProjectManager/Presentation/TodoListViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  TodoListViewModel.swift
+//  ProjectManager
+//
+//  Created by 이시원 on 2022/07/07.
+//
+
+import Foundation
+import RxSwift
+
+protocol TodoListViewModelInput {
+    var listItems: BehaviorSubject<[TodoMedel]> { get }
+}
+
+protocol TodoListViewModelOutput {
+    var todoList: Observable<[TodoMedel]> { get }
+    var doingList: Observable<[TodoMedel]> { get }
+    var doneList: Observable<[TodoMedel]> { get }
+}
+
+protocol TodoListViewModel: TodoListViewModelInput, TodoListViewModelOutput {}
+
+final class DefaultTodoListViewModel: TodoListViewModel {
+    let listItems = BehaviorSubject<[TodoMedel]>(value: TodoMedel.makeDummy())
```

Comment:
디버그 빌드에서만 돌아가도록 전처리문을 사용할 땐, 반드시 릴리즈 빌드도 정상적으로 동작하는지 확인해줘야 합니다.
현재 `makeDummy` 메서드의 선언부를 디버그 빌드에서만 컴파일되도록 해두었으나, 이곳은 따로 전처리문을 선언해두지 않아서 릴리즈 빌드에선 에러가 발생할 거예요.
```suggestion
    #if DEBUG
    let listItems = BehaviorSubject<[TodoMedel]>(value: TodoMedel.makeDummy())
    #else
    let listItems = BehaviorSubject<[TodoMedel]>(value: [])
    #endif
```

Reply:
감사합니다. 이런부분 까지 생각을 못했습니다. 수정 했습니다